### ISSUE TITLE
Multiclass Losses: Gibt es das Äquivalent zu Cross Entropy / Bernoulli Loss auf Scores wie im binären auch für Multiclass? 

### ISSUE THREAD

**Issue by juliambr** [slds-lmu/lecture_i2ml#916]
Können wir bei der Softmax regression - genauso wie bei der logistischen Regression - den loss auch auch auf scores definieren?

**Comment by lisa-wm** [comment]
BB: mathematical formulation should be possible but log of sum over exp (softmax denominator) won't lead us anywhere meaningful
